lib/dl_connector_ydb/dl_connector_ydb/locales/en/LC_MESSAGES/dl_formula_ref_dl_connector_ydb.po

```diff
@@ -9,5 +9,6 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
+#, python-brace-format
```

Comment:
Doesn't this mess up .mo files?

Reply:
frankly, have no idea what's that, just did `gen:i18n-po` `gen:i18n-binaries`